action.yml

```diff
@@ -36,77 +40,66 @@ outputs:
 runs:
   using: 'composite'
   steps:
+    - uses: actions/setup-node@v3
+    - run: yarn add neonctl
```

Comment:
Instead of adding `yarn`, could you use `npx`?

Reply:
what would be the difference? setup-node comes with both yarn and npm included

Reply:
Mostly https://github.com/neondatabase/create-branch-action/issues/9, it looks like yarn add tries to also check all the other dependencies in the project and may have incompatibilities.

Reply:
Will installing package globally via yarn help?
https://github.com/neondatabase/create-branch-action/pull/10/commits/6200abcc1702897f892cf31361df46a27056cbc0

Reply:
Seems reasonable, I can test it today or tomorrow.

Reply:
Thanks!